tinyxml2.h

```diff
@@ -443,7 +443,7 @@ class MemPoolT : public MemPool
 
     union Item {
         Item*   next;
-        char    itemData[ITEM_SIZE];
+        char    itemData[static_cast<size_t>(ITEM_SIZE)];

```

Comment:
likewise, shouldn't we change ITEM_SIZE?

Reply:
As above, tho this would mix signed and unsigned ints even more.